src/Faker/Provider/en_GB/Person.php

```diff
@@ -67,27 +67,30 @@ class Person extends \Faker\Provider\Person
     );
 
     /**
-     * @link http://surname.sofeminine.co.uk/w/surnames/most-common-surnames-in-great-britain.html
+     * @link https://surnames.behindthename.com/top/lists/england-wales/1991
```

Comment:
Unfortunately that source is copyrighted. We can't use it in an MIT project like Faker. Please find an open-source list, or close the PR.